csrankings-r.csv

```diff
@@ -391,6 +391,7 @@ Ravi Janardan,University of Minnesota,http://umn.edu/home/janardan,NOSCHOLARPAGE
 Ravi K. Iyer,Univ. of Illinois at Urbana-Champaign,https://www.ece.illinois.edu/directory/profile/rkiyer,NOSCHOLARPAGE
 Ravi Karkar,University of Massachusetts Amherst,https://homes.cs.washington.edu/~rkarkar,n_vk_EAAAAAJ
 Ravi Mukkamala,Old Dominion University,http://www.cs.odu.edu/~mukka,NOSCHOLARPAGE
+Ravi Netravali,Princeton University,https://www.cs.princeton.edu/~ravian,4C9naMgAAAAJ

```

Comment:
Please remove the corresponding entries from `old/industry.csv` - thanks.

Reply:
Done. Thanks for your patience.